packages/core/admin/admin/src/translations/en.json

```
@@ -181,15 +181,16 @@
   "Settings.permissions.auditLogs.action": "Action",
   "Settings.permissions.auditLogs.date": "Date",
   "Settings.permissions.auditLogs.user": "User",
+  "Settings.permissions.auditLogs.user.fullname": "{firstname} {lastname}",
   "Settings.permissions.auditLogs.userId": "User ID",
   "Settings.permissions.auditLogs.details": "Log Details",
   "Settings.permissions.auditLogs.payload": "Payload",
   "Settings.permissions.auditLogs.listview.header.subtitle": "Logs of all the activities that happened in your environment",
-  "Settings.permissions.auditLogs.entry.create": "Create entry ({model})",
-  "Settings.permissions.auditLogs.entry.update": "Update entry ({model})",
-  "Settings.permissions.auditLogs.entry.delete": "Delete entry ({model})",
-  "Settings.permissions.auditLogs.entry.publish": "Publish entry ({model})",
-  "Settings.permissions.auditLogs.entry.unpublish": "Unpublish entry ({model})",
+  "Settings.permissions.auditLogs.entry.create": "Create entry {model, select, undefined {} other {({model})}}",
```

Nitpicking a bit but I think the space after entry should be inside the condition, so that it only exists when model is given

---

I've added it to the next FE sync to discuss if we should use the https://formatjs.io/docs/core-concepts/icu-syntax#select-format Syntax more in these cases to avoid having multiple strings (based on a dynamic id). I think that would really help us to keep the number of translations down and our users because strings like that would become easier to translate:

```suggestion
  "Settings.permissions.auditLogs.entry.create": "{action, select, 
    create {Create}
    delete {Delete}
    ...
  } entry {model, select, undefined {} other {({model})}}",
```

---

@gu-stav I explored this here: https://github.com/strapi/strapi/pull/15657/files. I'm not completely convinced by the idea. At least for audit logs I feel like it made a lot of things unnecessarily complicated (or maybe that was just me 🤦‍♂️ ). 

What's the concern about having many relation keys? I think the dynamic id approach still has merit. I also just find the syntax a bit annoying to work with, especially long strings (ie multiple selects) that will end up on a single line. Anyway happy to discuss it further and the PR is there too to keep the conversation going. 👍